api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -98,6 +98,20 @@
 	public Order saveOrder(Order order, OrderContext orderContext) throws APIException;
 	
 	/**
+	* Save or update the given <code>order</code> in the database. If the OrderType for the order
+	* is not specified, then it will be set to the one set on the OrderContext if any, if none
+	* exists on the orderContext, then it will be set to the one associated to the ConceptClass of
+	* the ordered concept otherwise the save fails. If the CareSetting field of the order is not
+	* specified then it will default to the one set on the passed in OrderContext if any otherwise
+	* the save fails. This method is different from saveOrder(Order order, OrderContext orderContext) above: 
+	* param parallelOrders contains known overlapping orders to be bypassed in overlapping check, otherwise 
+	* AmbiguousOrderException will be thrown
+	* @should pass if an known drug order for the same drug formulation specified
+	*/
+	@Authorized( { PrivilegeConstants.EDIT_ORDERS, PrivilegeConstants.ADD_ORDERS })
+	public Order saveOrder(Order order, OrderContext orderContext, Order[]  parallelOrders) throws APIException;
```

Comment:
I think you should use the orderContext for now instead and not overload this method